alcs-api/src/main.ts

```diff
@@ -12,7 +12,7 @@ async function bootstrap() {
   // fastify
   const app = await NestFactory.create<NestFastifyApplication>(
     AppModule,
-    new FastifyAdapter({ logger: true }),
+    new FastifyAdapter({ logger: JSON.parse(config.get<string>('VERBOSE')) }),
```

Comment:
JSON.parse here seems a little odd, is there a reason this isn't a boolean in the config and get<boolean> here?

Reply:
initially I had issue with boolean, even like this:  config.get<boolean>('VERBOSE'). Anyway, it is not relevant anymore, since I've changed it to LOG_LEVEL instead